crates/runtime/src/task_history.rs

```diff
@@ -37,6 +37,8 @@ use crate::accelerated_table::{AcceleratedTable, Retention};
 pub mod otel_exporter;
 
 pub const DEFAULT_TASK_HISTORY_TABLE: &str = "task_history";
+pub const DEFAULT_TASK_HISTORY_RETENTION_PERIOD_SECS: u64 = 24 * 60 * 60; // 1 day
+pub const DEFAULT_TASK_HISTORY_RETENTION_CHECK_INTERVAL_SECS: u64 = 30 * 60; // 30 minutes
```

Comment:
Let's reduce this default.

```suggestion
pub const DEFAULT_TASK_HISTORY_RETENTION_CHECK_INTERVAL_SECS: u64 = 16 * 60; // 15 minutes
```